packages/bpk-component-phone-input/src/BpkPhoneInput.js

```diff
@@ -178,28 +177,27 @@ const BpkPhoneInput = (props: Props) => {
         </BpkSelect>
       </div>
       <div className={getClassName('bpk-phone-input__phone-number')}>
-        <BpkLabel
-          htmlFor={id}
-          disabled={disabled}
-          // TODO: className to be removed
-          // eslint-disable-next-line @skyscanner/rules/forbid-component-props
-          className={getClassName('bpk-phone-input__label')}
-        >
-          {label}
-        </BpkLabel>
-        {/* $FlowFixMe[cannot-spread-inexact] - inexact rest. See 'decisions/flowfixme.md'. */}
-        <BpkInput
-          {...commonProps}
-          {...rest}
-          id={id}
-          name={name}
-          value={displayValue}
-          type={INPUT_TYPES.tel}
-          onChange={handleChange}
-          // TODO: className to be removed
-          // eslint-disable-next-line @skyscanner/rules/forbid-component-props
-          className={getClassName(className)}
-        />
+        <div className={getClassName('bpk-phone-input__label')}>
+          <BpkLabel
+            htmlFor={id}
+            disabled={disabled}
+          >
+            {label}
+          </BpkLabel>
+        </div>
+          {/* $FlowFixMe[cannot-spread-inexact] - inexact rest. See 'decisions/flowfixme.md'. */}
+          <BpkInput
+            {...commonProps}
+            {...rest}
+            id={id}
+            name={name}
+            value={displayValue}
+            type={INPUT_TYPES.tel}
+            onChange={handleChange}
+            // TODO: className to be removed
+            // eslint-disable-next-line @skyscanner/rules/forbid-component-props
+            className={getClassName(className)}
+          />
```

Comment:
This seems to be indented by 1 too many.

Reply:
Oh yeah, thanks for spotting Matt, fixed now.